frontend/src/metabase/selectors/embed.ts

```diff
@@ -1,7 +1,7 @@
 import { isWithinIframe } from "metabase/lib/dom";
 import { State } from "metabase-types/store";
 
-export const getIsEmbedded = (state: State) => {
+export const getIsEmbedded = (_state?: State) => {
```

Comment:
Just came across this and fix the signature so that it doesn't require any parameter since it shouldn't.